tests/posix/eventfd/testcase.yaml

```diff
@@ -3,7 +3,44 @@ common:
   tags:
     - posix
     - eventfd
+  # 1 tier0 platform per supported architecture
   integration_platforms:
+    - mps2/an385
+    - native_sim
+    - native_sim/native/64
+    - qemu_arc/qemu_arc_em
+    - qemu_arc/qemu_arc_hs6x
+    - qemu_cortex_a53
+    - qemu_leon3
+    - qemu_malta
+    - qemu_riscv32
+    - qemu_riscv64
+    - qemu_xtensa
     - qemu_x86
+    - qemu_x86_64
 tests:
-  portability.posix.eventfd: {}
```

Comment:
Given that picolibc is usually the default, it looks like this configuration will test against picolibc twice and never test against minimal libc?

Reply:
It's been a while, but IIRC the reason why the base case exists and is empty is to allow people to override things easier on the command line. So, for example, if a completely external libc was used, people could still run the suite without having to n-select anything in addition to y-selecting things.

Reply:
I could add an explicit case for the minimal libc though. Good catch 👍

Reply:
Thanks, that makes good sense, it's nice to provide a way for people to test things outside of the "known" C libraries. I was mostly wondering if we want to test minimal libc at all, because aiui, it's not happening with the proposed change (I don't know about the pre-patch status).

Reply:
Went through all of them and made sure there was a `.minimal` case as well. So far so good on my desk, so I'll push.

```
INFO    - Added initial list of jobs to queue
INFO    - Total complete: 1822/2488  73%  skipped: 1341, failed:    0, error:    0
```